app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -334,6 +346,11 @@ class HomeFragment : Fragment() {
                 settings = components.settings,
                 engine = components.core.engine,
                 metrics = components.analytics.metrics,
+                messageController = DefaultMessageController(
```

Comment:
Tell me a controller needs to be refactored, without telling me a controller needs to be refactored. 😬 

<img width="598" alt="Screen Shot 2022-03-18 at 12 43 59 PM" src="https://user-images.githubusercontent.com/1370580/159046129-358c3977-5303-4e33-b3b8-f5e8c6e1dc9c.png">

Reply:
😄